Version bump for LevelGraph v2 compatibility?

I'm using both this and `levelgraph-jsonld` together on [LevelGraph Playground](http://github.com/bigbluehat/levelgraph-playground). However, this one's not quite ready for today's LevelGraph 2.x release.

NPM says:
```
npm ERR! peerinvalid The package levelgraph@2.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer levelgraph-jsonld@1.0.0 wants levelgraph@^2.0.0
npm ERR! peerinvalid Peer levelgraph-n3@1.2.0 wants levelgraph@>= 1.0 < 2.0
```

Maybe the `package.json` just needs a tweak?

/cc @jmatsushita

Thanks!
🎩

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
   - "6"
+  - "7"
 script:
   - npm run coverage
   - npm run publish-coverage

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "readable-stream": "^2.1.4"
   },
   "peerDependencies": {
-    "levelgraph": ">= 1.0 < 2.0"
+    "levelgraph": "^2.0.0"
   },
   "devDependencies": {
     "mocha": "^3.0.0",

--- a/package.json
+++ b/package.json
@@ -45,8 +45,8 @@
   },
   "devDependencies": {
     "mocha": "^3.0.0",
-    "levelgraph": "^1.0.0",
-    "level-test": "^2.0.0",
+    "levelgraph": "^2.0.0",
+    "memdb": "^1.3.1",
     "chai": "^3.5.0",
     "coveralls": "^2.11.2",
     "istanbul": "^0.4.4",

--- a/test/helper.js
+++ b/test/helper.js
@@ -1,5 +1,5 @@
 var fs       = require('fs'),
-    level    = require('level-test')(),
+    level    = require('memdb'),
     graph    = require('levelgraph'),
     n3   = require('../');
 

